commitlint.config.ts

```diff
@@ -226,7 +226,7 @@ module.exports = {
 
                     return [
                         !offence,
-                        `Please watch out for leading or ending trailing whitespace`
+                        `Please watch out for whitespaces before parentheses`
```

Comment:
@realmarv mmm, this message might be confusing to the user; think about this case! he writes the commit message `Frontend (SendPage): fix crash (thread issue)`, then she will remove all the spaces before parentheses; but we only care about the first one (because it is inside the scope).

Reply:
Done!